cmd/server/main.go

```diff
@@ -204,20 +193,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Initialize deduplication
-	if conf.Dedupe.Enabled {
-		deduplicator, err := conf.Dedupe.NewDeduplicator()
-		if err != nil {
-			logger.Error("failed to initialize deduplicator", "error", err)
-			os.Exit(1)
-		}
-
-		ingestCollector = ingest.DeduplicatingCollector{
-			Collector:    ingestCollector,
-			Deduplicator: deduplicator,
-		}
-	}
```

Comment:
Can you please explain the changes around deduplication? (Keep in mind that we may have to hold off on merging this or tagging a new openmeter, until we roll out the proxy removal in cloud to prod AND also need to make these changes in cloud)